### PR TITLE
SEO-182222-Maui-SignaturePad-External-linking

### PR DESCRIPTION
**Pull request by Edithabich**
|Title|Description|
|--|--|
|Task Category| .NET MAUI SignaturePad|
|Content Task Link/Mail Screenshot|  |
|UX task| NA|
|External linking PR link| External linking -https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-signature-pad/pull/4 |
|Ticket/Task link| https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/182222/ |
|Excel/SharePoint link| https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-Signature-Pad--Money-keywords.aspx |

| Keyword | Volume | Position | Reason for change | Changes Made |
| ---      | ---       | ---     | ---     | ---     |
|.net maui SignaturePad|0-10 |6UG|To improve our FT experience |added CSS classes, alignment format, renamed and aligned images in zigzag form, added documentation, and code example |
|.net maui SignaturePad example|0-10 |5GitHub|To improve page experience | demo link added, code example|
|.net maui SignaturePad control |0-10 |6GitHub|To improve our FT experience | improved on page changes|
|SignaturePad in .net maui |0-10|5GitHub |To improve FT ranking to 1st position| added CSS classes alignment format, images aligned in zigzag form, added documentation & Example links, unwanted space removed|

Worked on 18 keywords in the .NET MAUI SignaturePad and am keeping track of all keyword details in the link below:-
 https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-Signature-Pad--Money-keywords.aspx

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,5 @@
...
-
-Refer to the following documentation for the Syncfusion .NET MAUI SignaturePad Control: 
-https://help.syncfusion.com/maui/signaturepad/getting-started
+The [.NET MAUI SignaturePad](https://www.syncfusion.com/maui-controls/maui-signaturepad?utm_source=github&utm_medium=listing&utm_campaign=maui-signaturepad-github-samples) is an interactive UI control that allows users to capture smooth and realistic signatures. Also, users can save their signatures as images and sync them across devices and documents that need signatures. A quick-start project that helps you integrate the Signature Pad control with the .NET MAUI application by adding the Syncfusion.Maui.SignaturePad package from NuGet.
 
 ## Project prerequisites
 Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/ ) with the Dot NET MAUI workload and [.NET Core SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.
```
Comment (Sharly-Atieno) [inline README.md]: Remove extra space here
image

File: README.md [hunk 2]
```diff
@@ -1,8 +1,5 @@
 # Getting Started with the .NET MAUI SignaturePad 
-A quick-start project that helps you integrate the Signature Pad control with the .NET MAUI application by adding the Syncfusion.Maui.SignaturePad package from NuGet.
-
-Refer to the following documentation for the Syncfusion .NET MAUI SignaturePad Control: 
-https://help.syncfusion.com/maui/signaturepad/getting-started
+The [.NET MAUI SignaturePad](https://www.syncfusion.com/maui-controls/maui-signaturepad?utm_source=github&utm_medium=listing&utm_campaign=maui-signaturepad-github-samples) is an interactive UI control that allows users to capture smooth and realistic signatures. Also, users can save their signatures as images and sync them across devices and documents that need signatures. A quick-start project that helps you integrate the Signature Pad control with the .NET MAUI application by adding the Syncfusion.Maui.SignaturePad package from NuGet.
```
Comment (Sharly-Atieno) [inline README.md]: No need for this line
image

Comment (Edithabich) [reply README.md]: resolved

**Review by Sharly-Atieno — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**